draft-ietf-quic-qlog-main-schema.md

```diff
@@ -105,7 +105,7 @@ definitions).
 
 The goal of this high-level schema is to provide amenities and default
 characteristics that each logging file should contain (or should be able to
-contain), such that generic and reusable toolsets can be created that can deal
+contain), such that generic and reusable tool sets can be created that can deal
```

Comment:
this kinda looks fine as it is? but I'll leave it to you, I don't feel strongly

Reply:
This was my opinionated spellchecker, I'll revert.